runtime/jvmti/jvmtiCapability.c

```diff
@@ -274,11 +276,12 @@ jvmtiGetPotentialCapabilities(jvmtiEnv* env, jvmtiCapabilities* capabilities_ptr
 	if ((*vmHook)->J9HookIsEnabled(vmHook, J9HOOK_VM_RESOURCE_EXHAUSTED)) {
 		rv_capabilities.can_generate_resource_exhaustion_heap_events = 1;
 	}
-
+#if JAVA_SPEC_VERSION >= 9
 	if (JVMTI_PHASE_ONLOAD == jvmtiData->phase) {
 		rv_capabilities.can_generate_early_vmstart = 1;
 		rv_capabilities.can_generate_early_class_hook_events = 1;
 	}
+#endif
```

Comment:
Please write this as `#endif /* JAVA_SPEC_VERSION */` like you did above on line 95 (here and everywhere else you introduced conditional code).